xml/ha_storage_protection.xml

```diff
@@ -850,12 +850,12 @@ Timeout (msgwait)  : 10
     <varlistentry>
      <term>Predictable static delays</term>
      <listitem>
-      <para>This parameter enables a static delay before executing &stonith; actions.
-      It ensures that the nodes do not fence each other if separate fencing resources
-      and different delay values are being used. The targeted node will
-      loose in a fencing match.
-      The parameter can be used to <quote>mark</quote> a specific node to survive
-      in case of a split brain scenario in a two-node cluster.
+      <para>This parameter adds a static delay before executing &stonith; actions.
+      To prevent the nodes from getting reset at the same time under split-brain of
+      a two-node cluster, configure separate fencing resources with different delay values.
+      The targeted node will loose in a fencing match.
+      The parameter can be used to mark the preferred node to be targeted with a
+      longer fencing delay, so that it wins any fencing match.
```

Comment:
If we read the whole paragraph, would something like this sound more natural:

"The preferred node can be marked with the parameter to be targeted with a longer fencing delay, so that it wins any fencing match."

?

Reply:
Okay, thanks, integrated. :slightly_smiling_face: